clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -1677,6 +1678,42 @@ def parse_rcflow(flow):
 
     return counter_id, index
 
+def _format_dce(ce):
+    output_buffer = 'table_id={}'.format(ce.table_entry.table_id)
+    output_buffer += ', counter_data=(bytes={},packets={})'.format(ce.data.byte_count, ce.data.packet_count)
+    return output_buffer
+
+@with_client
+def p4ctl_get_direct_counter_entry(client, bridge, cnt_tbl_name, flow):
+    """
+    get-direct-counter-entry SWITCH COUNTER_TABLE READ_COUNTER_FLOW
```

Comment:
in the format on line 1689, it says **get-direct-counter-entry** but in example on line 1691, it says **get-direct-counter**

Reply:
line 1689 is added just as comments wrt to function name p4ctl_get_direct_counter_entry as get-direct-counter-entry
and "get-direct-counter" is CLI command